### ISSUE TITLE
LotPurpose is not linked to the Lot

### ISSUE THREAD

**Issue by csnyulas** [OP-TED/ted-rdf-mapping#185]
There seems to be no link between lot and purpose.

The additional codes for the lot are in the RDF, but no link to the purpose.  661127-2021

**Comment by csnyulas** [comment]
The problem is in the join condition (technical mapping). Related to the fix of issue #157.

**Comment by csnyulas** [comment]
This SPARQL query demonstrates that **all** `epo:Purpose`  instances are referenced (i.e. there is no `epo:Purpose` instance that has no property pointing to it).

```
SELECT ?type ?s ?p ?o
{
 ?o a epo:Purpose  .
 OPTIONAL { 
  ?s ?p ?o.
  OPTIONAL { ?s a ?type.} 
 }
}
ORDER BY ?type
```

A somewhat more advanced query shows that the relationships to the `epo:Purpose` instances are present both in case of single lot procedures and for procedures that have multiple lots.

```
SELECT ?type ?s ?proc ?p ?o
{
 ?o a epo:Purpose  .
 OPTIONAL { 
  ?s ?p ?o.
  OPTIONAL { ?s a ?type.} 
  OPTIONAL { ?proc epo:hasProcurementScopeDividedIntoLot ?s }
 }
}
ORDER BY ?type
```
